### PR TITLE
Database setup

### PR DESCRIPTION
**Pull request by MWeberLambdaweb19**
# Description

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [x] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

We reviewed the database publicly, but no tests are available for database setup.

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

**Comment by daveskull81** [comment]
[Set up database](https://trello.com/c/FY9lOnN6/6-set-up-database)

File: data/development/seeds/004-images.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+
```
Comment (Memitaru) [inline data/development/seeds/004-images.js]: For seeds we should have at least one example of multiple images for one piece of art just for testing purposes

File: knexfile.js [hunk 2]
```diff
@@ -26,19 +56,18 @@ module.exports = {
...
 
   production: {
-    client: 'postgresql',
-    connection: {
-      database: 'my_db',
-      user:     'username',
-      password: 'password'
-    },
+    client: 'pg',
+    connection: process.env.DATABASE_URL,
```
Comment (Memitaru) [inline knexfile.js]: Make sure you update the README with info on any environment variables

File: data/development/seeds/004-images.js (hunk 1 above)
Comment (MWeberLambdaweb19) [reply data/development/seeds/004-images.js]: We have updated the seeds file to reflect this request!

File: knexfile.js (hunk 2 above)
Comment (MWeberLambdaweb19) [reply knexfile.js]: We have updated the README with this info!

**Pull request merged**